source/release-notes/v1.8-release-notes.rst

```diff
@@ -102,38 +103,149 @@ The Open OnDemand Portal :ref:`ood-portal-generator-configuration` file now supp
 
+Batch connect apps can now enable or disable caching site wide, on a per app basis
+or on a per item basis.
+
+:ref:`See the full documentation on this feature here. <caching-from-items>`
+
 Extended view support for GridEngine clusters
 .............................................
 
-Torque and SLURM schedulers now supports QOS fields
+In the Active Jobs app, each job will have a dropdown where the user can see various details about their job such as walltime, Total CPUs, etc. 
+This functionality was recently implemented for SGE in v1.8 and corresponds to ood_core v0.12.0
+
+The data seen in extended view is extracted from the command ``qstat -j -r -xml``.
+
+Torque and Slurm schedulers now support QoS fields
 ...................................................
 
+Torque and Slurm schedulers now support the quality of service (QoS) fields.
+
+``qos`` is now an acceptable script attribute like so:
+
+.. code-block:: yaml
+
+   script:
+     qos: 'best'
+
+In Torque this translates to ``-l qos=best`` and in Slurm it is ``--qos best``.
+
 Shell hosts are now protected behind allowlists
 ...............................................
 
+Shell hosts are now protected behind allowlists. This means administrators now need
+to explicitly add hosts to an allowlist for users to be able to access them through
+the shell app.
+
+Any currently configured login host that is not hidden will be automatically allowed.
+However, administrators will need to add a configuration for compute nodes and any
+additional hosts they wish to allow access to.
+
+Administrators should also configure the default ssh host if they've not already done
+so because 'localhost' is no longer the out of the box default.
+
+:ref:`See the documentation for setting the default ssh host <set-default-ssh-host>`
+and :ref:`the documentation for setting the allowlist for more details <set-ssh-allowlist>`.
+
 Beta XDMOD integration in the dashboard
 .......................................
 
+If your site also uses `XDMOD`_ for job metric collection, Open OnDemand now
+has beta support to connect to that installation and show XDMOD panels in
+the dashboard.
+
+To enable this functionality simply set the ``OOD_XDMOD_HOST`` to it's full URL
+in ``/etc/ood/config/apps/dashboard/env``.
+
+.. code-block:: shell
+
+  OOD_XDMOD_HOST="https://xdmod.mysite.edu"
+
 Chinese localization
 ....................
 
+The Chinese localization ``zh-CN`` is now available in the dashboard and
+job composer applications for sites who wish to use it.
+
+To enable it, simply set the ``OOD_LOCALE`` environment variable globally
+in ``/etc/ood/config/nginx_stage.yml``
+
+.. code-block:: yaml
+
+  pun_custom_env:
+     OOD_LOCALE: "zh-CN"
+
+
 Ability to add additional info to batch connect app
 ...................................................
 
-Support for SVG logos
-.....................
+Batch connect app developers may now add files that add information the the batch
```

Comment:
You can now add an info.html.erb file to an interactive app (batch connect) directory and it will render and display in the job panel. This is useful for providing documentation links for an app. :ref:`See the documentation on info.html.erb for more details <bc_info_html_md_erb>`.